ils_middleware/tasks/amazon/alma_instance_s3.py

```diff
@@ -18,38 +17,44 @@ def send_instance_to_alma_s3(**kwargs):
         instance_uri = URIRef(instance_uri)
         work_uri = None
         instance_graph = Graph()
-        work_graph = Graph()
         instance_graph.parse(instance_uri)
+
         # Define the bf namespace
         bf = Namespace("http://id.loc.gov/ontologies/bibframe/")
+
+        # Bind the namespaces to the instance graph
         for prefix, url in alma_namespaces:
             instance_graph.bind(prefix, url)
-        work_uri = instance_graph.value(
-            subject=URIRef(instance_uri), predicate=bf.instanceOf
-        )
-        if work_uri is None:
-            logger.info(f"Instance {instance_uri} has no work.")
-            continue
+
+        # Get the work URI from the instance graph
+        work_uri = instance_graph.value(subject=instance_uri, predicate=bf.instanceOf)
```

Comment:
I think you'll want to check to see if work_uri is None, otherwise you'll raise a TypeError on line 33.

Reply:
thanks, I'll add that check!